config/iam/cas/activeeon-private.key

```diff
@@ -0,0 +1,15 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Why are we publishing a private key ?

Reply:
The user can configure IAM to sign and encrypt some information (e.g., JWT Json Web Token, credentials)  that will be sent to IAM clients. We use a private and a public key to sign and encrypt such information.

Reply:
I think that a private key published on github is not really a private anymore.

Reply:
I agree, and this is why the user has to change the keys in production. The ones in github are just for demo purposes.